### PR TITLE
remove canDebug context key

### PR DESCRIPTION
**Pull request by DavidKutu**
For #7056

-   [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -480,6 +480,7 @@
     "DataScience.interactiveWindowModeBannerSwitchAlways": "Always",
     "DataScience.interactiveWindowModeBannerSwitchNo": "No",
     "DataScience.ipykernelNotInstalled": "IPyKernel not installed into interpreter {0}",
+    "DataScience.needIpykernel6": "Ipykernel 6 is needed for debugging, click Install to continue. Or you can run 'pip install ipykernel==6.0.3/conda install ipykernel=6'",
```
Comment (rchiodo) [inline package.nls.json:483]: Is there a better command line to recommend? I feel like pip install --upgrade ipykernel would work too? Or maybe that doesn't always work?

Comment (rchiodo) [reply package.nls.json:483]: But at least the 6.0.3 version seems rather specific

**Review by rchiodo — APPROVED** [review]

File: package.nls.json (hunk 1 above)
Comment (DavidKutu) [reply package.nls.json:483]: I tested this a lot. Unfortunately, --upgrade didn't upgrade mine from 5.5.5 to 6

Comment (DavidKutu) [reply package.nls.json:483]: I have an issue on the python extension to fix the installer. If that's fixed, I'll remove the commands from the message.
https://github.com/microsoft/vscode-python/issues/16937

**Comment by roblourens** [comment]
How does this know not to show the RBL button on non-Python notebooks?

**Pull request merged**